unit/serialize.go

```diff
@@ -58,6 +58,31 @@ func Serialize(opts []*UnitOption) io.Reader {
 	return &buf
 }
 
+func SerializeSections(sections []*UnitSection) io.Reader {
+
+	var buf bytes.Buffer
+
+	if len(sections) == 0 {
```

Comment:
I think we don't need this, as an empty loop will have the same result.

Reply:
Will remove.